packages/lib/bookingSuccessRedirect.ts

```diff
@@ -24,6 +24,7 @@ export const bookingSuccessRedirect = async ({
 
     // Using parent ensures, Embed iframe would redirect outside of the iframe.
     window.parent.location.href = url.toString();
+    return;
```

Comment:
It was a missing return statement.
location.href doesn't immediately abort the control flow and it is followed by router.push. It seems to work on android and mac os but location.href get's ignored in iOS.